server/libs/domain/src/asset/response-dto/asset-response.dto.ts

```diff
@@ -32,13 +31,12 @@ export function mapAsset(entity: AssetEntity): AssetResponseDto {
   return {
     id: entity.id,
     deviceAssetId: entity.deviceAssetId,
-    ownerId: entity.userId,
+    ownerId: entity.ownerId,
     deviceId: entity.deviceId,
     type: entity.type,
     originalPath: entity.originalPath,
     resizePath: entity.resizePath,
     createdAt: entity.createdAt,
-    modifiedAt: entity.modifiedAt,
```

Comment:
This is the file-level modified time. Perhaps you think it was the database-related modified time?

Reply:
Yep, missed that, have changed the file related ones to fileModifiedAt and fileCreatedAt, and have also added createdAt to go alongside updatedAt like we have for most entities.